src/botPage/view/blockly/blocks/tools/webhook.js

```diff
@@ -0,0 +1,59 @@
+import { translate } from '../../../../../common/i18n';
+import { expectValue } from '../shared';
+
+Blockly.Blocks.webhook = {
+    init() {
+        this.jsonInit({
+            message0: translate('Webhook URL : %1'),
+            args0   : [
+                {
+                    type: 'input_value',
+                    name: 'WEBHOOK_URL',
+                },
+            ],
+            message1: translate('Payload : %1'),
```

Comment:
You can remove the space after `Payload`

```suggestion
            message1: translate('Payload: %1'),
```